src/screens/HomeScreen/components/SearchCarousel/SearchCarousel.tsx

```diff
@@ -0,0 +1,65 @@
+import React, {useState} from 'react';
+import {StyleSheet, View} from 'react-native';
+import Dots from 'react-native-dots-pagination';
+import {Carousel} from '../../../../components';
+import {useThemeColor} from '../../../../hooks';
+import {DeviceUtils, ItemLocaleUtils, verticalScale} from '../../../../utils';
+import {
+  SearchCarouselRenderItemProps,
+  SearchCarouselType,
+} from './SearchCarousel.types';
+import {SearchCarouselItem} from './SearchCarouselItem';
+
+export const SearchCarouselRenderItem = ({
+  index,
+  item,
+}: SearchCarouselRenderItemProps) => {
```

Comment:
Instead of creating a new type called SearchCarouselRenderItemProps, we should make use of the CarouselRenderItem<T> provided by carousel